src/components/Footer/Footer.tsx

```diff
@@ -84,13 +85,14 @@ const Footer = () => {
                                 color: colors.white,
                                 marginBottom: 0,
                                 marginTop: vw?.sm ? "64px" : "0",
-                                textAlign: vw?.sm ? "center" : "left",
+                                textAlign: "center",
                             }}
                         >
                             {t("footer:copyright")}
                         </h3>
                     </Row>
                 </Col>
+                <AletheiaInfo />
```

Comment:
**suggestion:** the `<Col>` component should be instiated here. The grid structure should remain in the same level, if you move it to the component `<AletheiaInfo />` we lose track of it's configurability.